src/commcare_cloud/ansible/roles/ebsnvme/tasks/main.yml

```diff
@@ -1,24 +1,32 @@
-- name: Install nvme-cli
-  become: yes
-  apt: name=nvme-cli state=present
-  notify: "Reload udev rules"
+- block:
+  - name: Install nvme-cli
+    become: yes
```

Comment:
Fair point! I might do this refactor when I get back to testing the full run-through on a machine type that uses this task book. (I'm trying not to incorporate changes without testing them manually, even if they're pretty easy to eyeballl, so that I don't introduce silly mistakes that pop up as errors later.)